hnn_core/cell.py

```diff
@@ -348,6 +348,11 @@ class Cell:
         by synapse type (keys can be soma_gabaa, soma_gabab etc.).
         Must be enabled by running simulate_dipole(net, record_isec=True)
         or simulate_dipole(net, record_isoma=True)
+    ca : dict
+        Contains recording of section speicifc calcium concentration. 
+        Must be enabled by running simulate_dipole(...) with record_ca
```

Comment:
```suggestion
        Must be enabled by running simulate_dipole(net, record_ca=True)
```

Reply:
It's not True, it's either 'all' or 'soma'

Reply:
I think @jasmainak is copying that from the docstring above for `record_isoma`, which is out of date.

But yes the docstring should read `Must be enabled by running simulate_dipole(..., record_ca='all')`

Reply:
I just wanted to say that use the function signature rather than using words to explain it.